src/world/World.ts

```diff
@@ -405,6 +405,19 @@ export default class World {
         chunk.addEntity(entity as any);
     }
 
+    /**
+     * Returns all entities (including players)
+     *
+     * You can filter this by either using the entity.getType() or
+     * entity.isPlayer() functions.
+     */
+    public getEntities(): Entity[] {
```

Comment:
We probably shouldn't be separating players & entities since a player is an entity. Not super relevant to this PR however.